package.json

```diff
@@ -2,7 +2,7 @@
   "name": "capture-app",
   "homepage": ".",
   "version": "35.0.0",
-  "cacheVersion": "1",
+  "cacheVersion": "2",
```

Comment:
I had to add some data to the cache. Increasing the cache version ensures the indexedDB cache is rebuilt.